app/views/chambers/peerage/admin/offline_poll_submitters/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row justify-content-center">
+  <div class="col-md-8">
+    <h1>Submit Offline <%=peerage.capitalize %> Poll</h1>
+    <%= bootstrap_form_for @ballot, url: chambers_admin_offline_poll_submitter_path do |f| %>
+      <%= f.select :peer_id, @ballot.peers.map{|p| [p.sca_name, p.id] }, {label: 'Peers', include_blank: true}, {class: 'selectpicker', data: {'live-search' => true}, required: true} %> 
+      <h3>Vote List</h3>
+      <%= f.fields_for :vote_list_advisings do |adv_field| %>
```

Comment:
Consider `f.fields_for :vote_list_advisings.select { |a| a.object.candidate.vote_list }` or something like that here, and the opposite below. The fields should ultimately have the same names (you can verify that in the generated HTML)